plug.vim

```diff
@@ -1089,7 +1089,7 @@ function! s:job_handler(job_id, data, event) abort
 
   if a:event == 'stdout'
     let complete = empty(a:data[-1])
-    let lines = map(filter(a:data, 'len(v:val) > 0'), 'split(v:val, "[\r\n]")[-1]')
+    let lines = map(filter(a:data, 'len(substitute(v:val, "[\r\n]", "", "g")) > 0'), 'split(v:val, "[\r\n]")[-1]')
```

Comment:
That works too, and even handles filtering the empty string so `len` isn't needed anymore.